js/scenes/Settings.js

```diff
@@ -45,13 +50,26 @@ export default class Settings extends Component {
           </View>
         ))}
 
+        {debugOptions}
+
         <TouchableOpacity
           onPress={this.onBackPress}>
           <Text style={styles.text}>Back</Text>
         </TouchableOpacity>
       </View>
     );
   }
+
+  renderDebug() {
+    if (__DEV__) {
+      return (<TouchableOpacity
+          onPress={this.onNavigate.bind(this, 'debug')}>
+          <Text style={styles.text}>Go to Debug</Text>
+        </TouchableOpacity>);
+    } else {
+      return '';
+    }
```

Comment:
Well it's essentially a `guard`. If we're not in DEV mode it's a noop. Francisco style:

```js
renderDebug() {
  if (!__DEV__) {
    return;
  }

  return <TouchableOpacity onPress={this.onNavigate.bind(this, 'debug')}>
     <Text style={styles.text}>Go to Debug</Text>
  </TouchableOpacity>;
}
```